example-get-started/code/README.md

```diff
@@ -57,12 +57,14 @@ $ dvc pull
 
 ## Running in your environment
 
-Run [`dvc repro`](https://man.dvc.org/repro) to reproduce the
-[pipeline](https://dvc.org/doc/commands-reference/pipeline):
+Run [`dvc exp run`](https://man.dvc.org/exp/run) to reproduce the
+[pipeline](https://dvc.org/doc/user-guide/pipelines) and create a new
+[experiment](https://dvc.org/doc/user-guide/experiment-management).
 
 ```console
-$ dvc repro
-Data and pipelines are up to date.
+$ dvc exp run
+Ran experiment(s): rapid-cane
+Experiment results have been applied to your workspace.
```

Comment:
No strong opinion on which we recommend here, but we still use `dvc repro` through the docs.

Reply:
yes, I plan to change docs also when this is deployed - wdyt? (dvc repro becomes more of a low level command)

Reply:
I go back and forth 😄 . Now I'm slightly in favor of keeping `repro` in this trail. Since we add one stage at a time, `dvc exp run` is a bit awkward to run after each stage is added. It's also unnecessarily heavy and the name doesn't fit the initial workflow we introduce very well. 

Most importantly, I don't think it is that impactful either way, so not something worth blocking.

Reply:
okay, yes. I guess you are right. It can be ultimately a mix `dvc repro` to do iterations on the pipeline, but `dvc exp run` when everything is ready for that, wdyt?

Reply:
I'm not sure there's that much benefit to introduce a new command there, but it might make sense there, or as a "next step." Again, not a blocker for me, and probably better to discuss in a docs PR anyway where we can see how it actually looks.